app/static/js/slcManagement.js

```diff
@@ -140,3 +140,54 @@ function unapproveProposal(el){
       }
     })
 }
+
+function updateInstructorList() { // This function fetches instructor usernames and attached the list of usernames to the form submission
+
+  $('#alterModal form input[name="instructor[]"]').remove(); // Clear existing hidden instructor inputs
+  var instructordata = getCourseInstructors(); // Get current instructors from the table
+
+  // Append new hidden inputs for each instructor
+  instructordata.forEach(function(instructor) {
+      $('<input>').attr({
+          type: 'hidden',
+          name: 'instructor[]',
+          value: instructor
+      }).appendTo('#alterModal form');
+  });
+}
+
+function updateInstructorsTable(instructors) { // This function creates row(s) for preexisting instructor(s) in the modal
+  // Clear existing table contents except the template row
+  $("#instructorTableBody").find("tr:not(:first)").remove();
+
+  // Add each instructor to the table
+  instructors.forEach(function(instructor) {
+    var newRow = createInstructorRow(instructor);
+    $("#instructorTableBody").append(newRow);
+  });
+}
+
+function createInstructorRow(instructor) {
+  // Create a new row element based on the instructor data
+  var row = `<tr data-username="${instructor.username}">
```

Comment:
I think the variable name 'row' should be more specific to what the row contains. It looks to be containing instructors' information, so maybe something that specifies that.